variants/NUCLEO_F207ZG/stm32f2xx_hal_conf.h

```diff
@@ -175,7 +175,7 @@
 
 /* Section 2: PHY configuration section */
 
-/* DP83848 PHY Address*/ 
+/* DP83848 PHY Address*/

```

Comment:
You're mixing up 2 things in the commit - clean-up of various white spaces with the actual changes for RTC.
In such cases, please try to make 2 separate commits instead ...

Reply:
Done!